bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Arena.java

```diff
@@ -113,7 +113,7 @@ public class Arena implements IArena {
 
 
     private List<Player> players = new ArrayList<>();
-    private List<Player> spectators = new ArrayList<>();
+    private final List<Player> spectators = new ArrayList<>();
```

Comment:
can you explain the changes in Arena class? thx

Reply:
@ImYenil

Reply:
do you know what final means?